src/frontend/components/UI/UninstallModal/index.tsx

```diff
@@ -22,7 +22,8 @@ const UninstallModal: React.FC<UninstallModalProps> = function (props) {
   const { handleGameStatus, platform } = useContext(ContextProvider)
   const [isWindowsOnLinux, setIsWindowsOnLinux] = useState(false)
   const [winePrefix, setWinePrefix] = useState('')
-  const [checkboxChecked, setCheckboxChecked] = useState(false)
+  const [prefixcheckboxChecked, setprefixCheckboxChecked] = useState(false)
+  const [settingcheckboxChecked, setsettingCheckboxChecked] = useState(false)
```

Comment:
Split the old checkboxChecked into prefixcheckboxChecked and settingcheckboxChecked